src/status_im/utils/instabug.cljs

```diff
@@ -3,6 +3,38 @@
             [status-im.utils.config :as config]
             [status-im.react-native.js-dependencies :as rn-dependencies]))
 
+(def instabug rn-dependencies/instabug)
+
+;; `event` is an event name, e.g. "Tap"
+;; `properties` is a map of event details or nil, e.g. {:target :send-current-message}
+;; (see mixpanel_events.edn for list of trackable events)
+
+(def survey-triggers
+  [{:event :send-current-message :count 4  :token "UqtvIKgVDUTo4l_sDS-fwA"}
+   {:event :send-current-message :count 29 :token "Hr9Dk3krPK7PPxuDbHAmXg"}])
```

Comment:
What is this token? Is it ok to just leave it open in the code like this?

Reply:
@goranjovic each token corresponds to particular survey created on Instabug.

@chadyj it shows "Messaging UX" after 5 messages
and "Confidence Test Survey April 2018" after 30 messages (don't look at numbers 4 and 29 in the code, it's 5 and 30 actually)

Reply:
@dmitryn can we reset "Confidence Test Survey ..." to trigger after 5 messages.

Reply:
@jpbowen yes, we will change it to 5 after "Messaging UX" survey will be finished.